src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -85,8 +81,6 @@ public void clearTrajectory() {
 
     /**
      * Gets the sendable chooser for Auton Modes
-     *
-     * @return
      */
     public SendableChooser<AutonChoices> getAutoChooser() {
```

Comment:
Put this doc comment back and complete the return comment since this has a return